test/functional/services/remote/webdriver.ts

```diff
@@ -79,6 +80,10 @@ async function attemptToCreateCommand(log: ToolingLog, browserType: Browsers) {
           // See: https://chromium.googlesource.com/chromium/src/+/lkgr/headless/README.md
           chromeOptions.push('headless', 'disable-gpu');
         }
```

Comment:
I've added this so you can remotely debug (view with devtools) the session as it's running headlessly